src/main/java/serenitylabs/tutorials/vetclinic/collections/katas/tasks/CheckIn.java

```diff
@@ -0,0 +1,48 @@
+package serenitylabs.tutorials.vetclinic.collections.katas.tasks;
+
+
+import net.serenitybdd.core.steps.Instrumented;
+import net.serenitybdd.screenplay.Actor;
+import net.serenitybdd.screenplay.Performable;
+import net.serenitybdd.screenplay.Task;
+import net.thucydides.core.annotations.Step;
+import org.jruby.ir.instructions.Instr;
+import serenitylabs.tutorials.vetclinic.Pet;
+import serenitylabs.tutorials.vetclinic.collections.katas.BookingResponse;
+import serenitylabs.tutorials.vetclinic.collections.katas.PetHotel;
+
+/**
+ * Created by vdheer on 9/20/2016.
+ */
+public class CheckIn implements Performable {
+    private final Pet pet;
+    private final PetHotel petHotel;
+
+    public CheckIn(Pet pet, PetHotel petHotel) {
+        this.pet = pet;
+        this.petHotel = petHotel;
+    }
+
+    @Step("{0} checks-in #pet into #petHotel")
+    @Override
+    public <T extends Actor> void performAs(T actor) {
+        BookingResponse bookingResponse = petHotel.checkIn(pet);
```

Comment:
Here, you never use the 'bookingResponse' variable, so you never actually check the response coming back from the hotel.